service-netty/src/main/java/info/bitrich/xchangestream/service/netty/NettyStreamingService.java

```diff
@@ -202,7 +202,6 @@ protected void initChannel(SocketChannel ch) {
                                         completable.onComplete();
                                     } else {
                                         webSocketChannel.disconnect().addListener(x -> {
-                                            scheduleReconnect();
```

Comment:
Nice! Just so I can follow the logic, which is the second reconnect that we hit?

Reply:
https://github.com/bitrich-info/xchange-stream/blob/6c2486cd3429e2eb15f76fec334318bd5c119e24/service-netty/src/main/java/info/bitrich/xchangestream/service/netty/NettyStreamingService.java#L435